presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -167,7 +167,7 @@ Specifier Description
 ``%D``    Day of the month with English suffix (``0th``, ``1st``, ``2nd``, ``3rd``, ...)
 ``%d``    Day of the month, numeric (``00`` .. ``31``)
 ``%e``    Day of the month, numeric (``0`` .. ``31``)
-``%f``    Microseconds (``000000`` .. ``999999``)
+``%f``    Nanoseconds (``000000000`` .. ``999999999``)
```

Comment:
I checked MySQL docs before I pushed this PR and they refer to _%f_ as "microseconds". I'd say _fraction_ is maybe better though.
Joda is referring to it as a _fractionOfSecond_. And frankly I like it.
What about calling it _fraction of second_ + adding a _.note_ section for it explaining what's the connection to MySQLs _microseconds_?